src/cmd/integration.go

```diff
@@ -229,9 +229,26 @@ EOF
 	},
 }
 
+var reactivateIntegrationCmd = &cobra.Command{
+	Use:        "reactivate ID",
```

Comment:
```suggestion
	Use:        "reactivate ID|ALIAS",
```

Reply:
I actually started with this then found out that integrations don't have aliases

Reply:
Ahh 😢 

But they still support `IdentifierInput` on the way in?